src/main/java/seedu/address/model/bid/Bid.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.bid;
+
+import java.util.Objects;
+
+public class Bid {
+
+
+    private String propertyId;
+    private String bidderId;
+    private double bidAmount;
+
+    /**
+     * Constructor for Bid Object
+     * @param propertyId string id of the property to bid for
+     * @param bidderId string id of the bidder wanting the property
+     * @param bidAmount double value of the amount the bidder wants the property for
+     */
+    public Bid(String propertyId, String bidderId, double bidAmount) {
+        this.propertyId = propertyId;
+        this.bidderId = bidderId;
+        this.bidAmount = bidAmount;
+    }
+
+    public String getPropertyId() {
+        return propertyId;
+    }
+
+    public String getBidderId() {
+        return bidderId;
+    }
+
+    public double getBidAmount() {
+        return bidAmount;
+    }
+
+    public boolean isValidPropertyId() {
+        return false; //will check against propertyList to see if propertyId exists
```

Comment:
I think it may be easier to check at the BidBook / UniqueBidList level (or even AddBidCommand, I'm not sure where is the best place yet), so that Bid does not know about PropertyBook. Maybe can think about this further in the future, but for now it's okay since other stuff is not done yet.